android/app/src/main/java/com/comapeo/WifiPackage.kt

```diff
@@ -0,0 +1,16 @@
+package com.comapeo
```

Comment:
See the [relevant React Native docs](https://reactnative.dev/docs/next/native-modules-android?android-language=kotlin#register-the-module-android-specific).